tools/build_rules/closure/closure_js_binary.bzl

```diff
@@ -46,6 +48,21 @@ _COMPILATION_LEVELS = {
   "advanced": ["--compilation_level=ADVANCED"]
 }
 
+_SUPPORTED_LANGUAGES = {
+  "es3": ["ES3"],
```

Comment:
Why support name aliases? Why not require that users enter the canonical name, which is the behavior of the Closure Compiler CLI.